_includes/popup.html

```diff
@@ -3,8 +3,8 @@
     <div class="picture">
     {% if include.person.image %}
     {% if include.who == 'student' %}
-    <img src="images/students/{{include.person.image}}"
-    alt="{{include.person.name}}"
+    <img src="images/students/HarshitBagla.jpg"
```

Comment:
Please don't add yourself using `index.html`.
You are to make changes by adding yourself to `.yml` files in `_data/` folder.
Please read the instructions in **Readme.md** for more info.